app/controllers/items_controller.rb

```diff
@@ -68,7 +68,7 @@ def set_item
     end
 
     # Never trust parameters from the scary internet, only allow the white list through.
-    def item_params
+    def item_params(c = 10, *d, a, b)
```

Comment:
__[rubocop]__
* Lint/UnusedMethodArgument: Unused method argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used. You can also write as `item_params(*)` if you want the method to accept any arguments but don't care about them. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Style/OptionalArguments: Optional arguments should appear at the end of the argument list. [:ref](https://github.com/bbatsov/ruby-style-guide#optional-arguments)
* Lint/UnusedMethodArgument: Unused method argument - `d`. If it's necessary, use `_` or `_d` as an argument name to indicate that it won't be used. You can also write as `item_params(*)` if you want the method to accept any arguments but don't care about them. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Lint/UnusedMethodArgument: Unused method argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used. You can also write as `item_params(*)` if you want the method to accept any arguments but don't care about them. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Lint/UnusedMethodArgument: Unused method argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used. You can also write as `item_params(*)` if you want the method to accept any arguments but don't care about them. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)